tests/queries/0_stateless/02000_select_with_filter.sql

```diff
@@ -0,0 +1,4 @@
+SELECT argMax(number, number + 1) FILTER(WHERE number != 99) FROM numbers(100) ;
+SELECT sum(number) FILTER(WHERE number % 2 == 0) FROM numbers(100);
+SELECT sumIfOrNull(number, number % 2 == 1) FILTER(WHERE number % 2 == 0) FROM numbers(100); -- { clientError 184 }
+SELECT sum(number) FILTER(WHERE) FROM numbers(100); -- { clientError 62 }
```

Comment:
There are some issues with AST fuzzer if we have unparsable queries in tests. But Actually first one have correct syntax, but we mustn't throw exception from parser